.rubocop.yml

```diff
@@ -44,6 +44,7 @@ AllCops:
   Exclude:
     - 'vendor/**/*'
     - 'db/**/*'
+    - 'app/queries/balance_query.rb'
```

Comment:
nit:
I would prefer to exclude particular cops instead of blacklisting whole file:
https://rubocop.readthedocs.io/en/latest/configuration/#disabling-cops-within-source-code